src/scripts/main.js

```diff
@@ -1 +1,19 @@
 'use strict';
+
+window.addEventListener('hashchange', () => {
+  if (window.location.hash === '#menu') {
+    document.body.classList.add('page__body--with-menu');
+  } else {
+    document.body.classList.remove('page__body--with-menu');
+  }
+});
+
+document.getElementById('form').addEventListener('submit',
+  function(e) {
+    e.preventDefault();
+
+    document.getElementById('name').value = '';
+    document.getElementById('email').value = '';
+    document.getElementById('message').value = '';
```

Comment:
```suggestion
    document.getElementById('form').reset();
```
Also, I think it's better to get it into a variable. const form = document.getElementById('form');

Reply:
спасибо, все замечания исправлены.